CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-ssm-mongodb ChangeLog
 
+## 10.2.0 - 2023-04-xx
+
+### Changed
```

Comment:
My read of semver is that this doesn't add any new API surface (nor any new dependency surface) so it's considered a "fix" that simplifies the implementation internally and should therefore be a patch.

```suggestion
## 10.1.3 - 2023-04-xx

### Fixed
```